README.md

```diff
@@ -48,13 +50,15 @@ town = {
 ```
 
 Using Ruby...
+
 - Remove Belle from the town residents
 - Add her to the list of guests in the castle.
 
 Write your code here...
 
 ```ruby
-# code here
+town[:residents] = ["Maurice, Gaston"]
+town[:guests] = "Belle"
 ```
```

Comment:
Your second line creates a new key `guests` that has a value of "Belle". The existing `guests` array is still there and still empty.

Here's a solution

```rb
town[:residents].delete("Belle")
town[:castle][:guests] << "Belle"
```